### DISCUSSION TITLE
Shelly plus plug s & Tasmota & Berry ... energy reading

### DISCUSSION THREAD

**Discussion by jan0wi** [arendst/Tasmota#18896] [Q&A]
Hello all, 

I am new in this universe, but I got stuck ...

Tried to convert a shelly plus plug s to tasmota with LED control (WS 1812) by energy meter. 
trying berry's energy.read() command gives a long json string with always "0".
Is there another approach to read out energy values to berry ? 

best regards 
Jan

**Comment by s-hadinger** [comment]
Please use the latest version in the development branch (not release). There was a hug a few weeks ago and it's maybe present in yiur version

**Reply by jan0wi** [reply]
I tried and it really works out of the box with tasmota 12.5.0.4 ; thanks a lot for your help !!
